buildkite/src/Entrypoints/GenerateHardforkPackage.dhall

```diff
@@ -45,6 +51,8 @@ let Spec =
           , config_json_gz_url =
               "https://storage.googleapis.com/o1labs-gitops-infrastructure/devnet/devnet-state-dump-3NK4eDgbkCjKj9fFUXVkrJXsfpfXzJySoAvrFJVCropPW7LLF14F-676026c4d4d2c18a76b357d6422a06f932c3ef4667a8fd88717f68b53fd6b2d7.json.gz"
           , suffix = ""
+          , version = "\\\$MINA_DEB_VERSION"
```

Comment:
I'm wondering why do we need double-escaping. And if possible could we avoid it?